zipline/pipeline/loaders/blaze/core.py

```diff
@@ -729,25 +729,36 @@ def __new__(cls,
             expr,
             deltas,
             checkpoints,
-            odo_kwargs or {},
+            frozenset((odo_kwargs or {}).items()),
```

Comment:
Why the change from dict to frozenset? Is this just being paranoid about mutation? Are all the objects we pass as values to `odo_kwargs` hashable?

Reply:
They are currently hashable (bytes values, unicode keys). This is because we want the `__hash__` of `ExprData` to be by value, not identity.